internal/core/data/controller/http/event_test.go

```diff
@@ -266,6 +266,70 @@ func TestAddEvent(t *testing.T) {
 	}
 }
 
+func TestAddEventSize(t *testing.T) {
+
+	dbClientMock := &dbMock.DBClient{}
+
+	validRequest := testAddEvent
+
+	tests := []struct {
+		Name               string
+		MaxEventSize       int64
+		RequestContentType string
+		errorExpected      bool
+		ExpectedStatusCode int
+	}{
+		{"Valid - AddEventRequest JSON with valid event size", 25000, common.ContentTypeJSON, false, http.StatusCreated},
+		{"Valid - AddEventRequest JSON with valid event size", 0, common.ContentTypeJSON, false, http.StatusCreated},
+		{"Invalid - AddEventRequest JSON with negative evnet size", -1, common.ContentTypeJSON, true, http.StatusBadRequest},
```

Comment:
How about another error test where `MaxEventSize=10`  to tests the actual capability we are adding here.